more_itertools/more.py

```diff
@@ -128,13 +128,7 @@ def chunked(iterable, n, strict=False):
         >>> list(chunked([1, 2, 3, 4, 5, 6, 7, 8], 3))
         [[1, 2, 3], [4, 5, 6], [7, 8]]
 
-    If this is the case and *strict* is True, then :func:`chunked` will
-    raise an exception.
-
-        list(chunked([1, 2, 3, 4, 5, 6, 7, 8], 3, strict=True))
-        Exception raised:
-            Traceback (most recent call last):
-            ValueError: There are fewer items in the last chunk!
+    If *strict* is True, then :func:`chunked` raises a `ValueError`.
```

Comment:
should it not be `will raise` or `shall raise` instead of `raises`?

Reply:
I think it's an either/or situation -- I prefer "raises" because of concision.